static/js/main.js

```diff
@@ -62,6 +62,18 @@ var header = {
         var password = $("#password").val();
         var verify_password = $("#verify_password").val();
 
+        if ($("#user_id")) {
+            var reg = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Please document this regexp, i.e., add a detailed comment before the regexp explaining each part of the regexp and what it matches.

Reply:
@tmmurali Thanks for the review. I have used this regex by referring to this link https://stackoverflow.com/questions/46155/how-can-an-email-address-be-validated-in-javascript .
I will surely try to document it.

Reply:
@lohani2280 Another suggestion! Can you create a util function for validating email regular expression? This way we can reuse it in other places if required.